package.json

```diff
@@ -104,22 +105,22 @@
 		"postcss-extend": "^1.0.5",
 		"postcss-hover-media-feature": "^1.0.2",
 		"postcss-import": "^16.1.0",
-		"postcss-licensing": "^2.0.0",
-		"postcss-load-config": "^5.1.0",
+		"postcss-licensing": "^3.0.0",
+		"postcss-load-config": "^6.0.1",
```

Comment:
Validated postcss-licensing breaking change update because no licensing results have changed. 🥳 

Validated postcss-load-config breaking change because the config loading in the scripts are not failing or returning different results.